### DISCUSSION TITLE
win10 启动后访问返回404

### DISCUSSION THREAD

**Discussion by jim138482040** [AlistGo/alist#1462] [General]
日志如下：
failed to init aria2 client: Post "http://localhost:6800/jsonrpc": dial tcp [::1]:6800: i/o timeout
failed get aria2 version
github.com/alist-org/alist/v3/internal/aria2.InitAria2Client
	G:/alist-main/internal/aria2/aria2.go:32
github.com/alist-org/alist/v3/internal/aria2.InitClient
	G:/alist-main/internal/aria2/aria2.go:22
github.com/alist-org/alist/v3/internal/bootstrap.InitAria2.func1
	G:/alist-main/internal/bootstrap/aria2.go:10
runtime.goexit
	C:/Program Files/Go/src/runtime/asm_amd64.s:1594

启动情况：
![图片](https://user-images.githubusercontent.com/100178665/183115194-2046b5f5-f9ca-4baf-ae5a-32c0d87ea454.png)
网页截图：
![图片](https://user-images.githubusercontent.com/100178665/183114541-b1d96125-8402-4826-982f-d9c25eb33964.png)
已经下载aria2  1.36
请问大佬这是啥原因

**Comment by rakaki** [comment]
试下右键用管理员身份启动
